test/parallel/test-runner-run-coverage.mjs

```diff
@@ -137,13 +137,13 @@ describe('require(\'node:test\').run coverage settings', { concurrency: true },
 
     await it('should run while including and excluding globs', async () => {
       const stream = run({
-        files: [...files, fixtures.path('test-runner/invalid-tap.js')],
+        files: files,
```

Comment:
Isn't this change also changing the test behaviour?

Reply:
No because `invalid-tap.js` was already required by `files`